packaging/Makefile_packaging.mk

```diff
@@ -125,8 +130,7 @@ all: $(TARGETS)
 	xz -z $<
 
 _topdir/SOURCES/%: % | _topdir/SOURCES/
-	rm -f $@
-	ln $< $@
+	if [ ! -d "$@" ]; then rm -f "$@"; ln "$<" "$@"; fi
```

Comment:
doesn't $@ expand to all args? also what is $< ? online guides dont seem to specify that one

Reply:
$< is the first dependency in the list of dependencies.   It is replaced by make before Bash sees it.
$@ is the effective target of the make rule.  It is replaced by make before Bash sees it.
For some unknown reason the recipe for this rule is being logged as run after a successful make build with the "%" apparently replaced by an empty string, which causes the "rm" command to emit a diagnostic.
I have not worked out why that is happening, but this is an attempt to suppress the diagnostic from the logs.